aptos/nft_bridge/sources/register_chain.move

```diff
@@ -0,0 +1,166 @@
+module nft_bridge::register_chain {
+
+    use wormhole::u16::{Self, U16};
+    use wormhole::cursor;
+    use wormhole::deserialize;
+    use wormhole::vaa;
+    use wormhole::external_address::{Self, ExternalAddress};
+
+    use nft_bridge::vaa as nft_bridge_vaa;
+    use nft_bridge::state;
+
+    /// "NFTBridge" (left padded)
+    const NFT_BRIDGE: vector<u8> = x"00000000000000000000000000000000000000000000004e4654427269646765";
+
+    const E_INVALID_MODULE: u64 = 0;
+    const E_INVALID_ACTION: u64 = 1;
+    const E_INVALID_TARGET: u64 = 2;
+
+    struct RegisterChain has copy, drop {
+        /// Chain ID
```

Comment:
Since RegisterChain has drop, does this mean you can re-register the same chain multiple times since it will just drop the previous registration?

Reply:
the registrations are stored in a table, which we upsert to so it should allow re-registration (but not because of the drop here)

Reply:
Don't we normally not allow re-registration?

Reply:
yeah but it's not specified in the whitepaper, and this seemed like better behaviour